uxarray/helpers.py

```diff
@@ -251,8 +242,9 @@ def get_all_face_area_from_coords(x,
     num_faces = face_nodes.shape[0]
     area = np.zeros(num_faces)  # set area of each face to 0
 
-    for i in range(num_faces):
+    face_nodes = face_nodes.astype(np.int64)
```

Comment:
We should check if our data is already stored as an integer to avoid unnecessary conversions if our data is another integer (uint32, int32, uint64, etc). 

Can do something like this:
```Python
if not np.issubdtype(face_nodes.dtype, np.integer):
    face_nodes = face_nodes.astype(np.int64)    

```